docs/using/administering/debugging.rst

```diff
@@ -33,6 +35,46 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Operating System and version,
+* kernel version,
+* Docker info and version,
+* all Flocker log files.
+
+The following scripts can be used to gather this information on Ubuntu and CentOS.
+
+Save the appropriate script to a file and then run it as root
+
+Ubuntu
+^^^^^^
+
+:download:`flocker-log-export-ubuntu.sh`
+
+.. literalinclude:: flocker-log-export-ubuntu.sh
+   :language: sh
+
```

Comment:
You click on the `docs` build results below and then look through the build steps for `upload-html` and click on `build-9098`:

* http://build.clusterhq.com/results/docs/log-documentation-FLOC-2647-2/build-9098/using/administering/debugging.html#bug-reporting